libexec/craftctl

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec "$SNAP"/bin/python "$SNAP"/bin/craftctl "$@"
```

Comment:
I would put /snap/snapcraft/current/bin/craftctl in libexec/snapcraft/craftctl and have snapcraft update the shebang instead

Reply:
Updated to move the actual python script to `libexec/snapcraft/craftctl` and replace the shebang to python in snap current.